src/components/molecules/main/sidebar/SocialIcons.tsx

```diff
@@ -0,0 +1,27 @@
+import youtube from "../../../../assets/images/youtube_logo.svg";
+import instagram from "../../../../assets/images/instagram_logo.svg";
+import { SocialIconsBox } from "./SocialIconsStyles";
+import { SocialIconImage } from "../../../atom/main/sidebar/SocialIconStyles";
+
+const SocialIcons: React.FunctionComponent = () => {
+  const SocialItems = [
+    { src: `${youtube}`, alt: "Youtube", href: "https://www.youtube.com/@jeju_deers_official" },
+    {
+      src: `${instagram}`,
+      alt: "Instagram",
+      href: "https://www.instagram.com/jeju_deers_football/",
+    },
+  ];
+
+  return (
+    <SocialIconsBox>
+      {SocialItems.map((item, index) => (
+        <a key={index} href={item.href} target="_blank" rel="noopener noreferrer">
+          <SocialIconImage src={item.src} alt={item.alt} />
+        </a>
+      ))}
+    </SocialIconsBox>
+  );
+};
```

Comment:
map 함수 없이도 작성할 수 있을 것 같은데 어떻게 생각하시나요? 참고로 봐주세요!

```suggestion
const SocialIcons = () => {
  return (
    <SocialIconsBox>
      <SocialIcon src={youtube} alt="Youtube" href="https://www.youtube.com/@jeju_deers_official" />
      <SocialIcon
        src={instagram}
        alt="Instagram"
        href="https://www.instagram.com/jeju_deers_football/"
      />
    </SocialIconsBox>
  );
};
```